src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/services/GoalService.kt

```diff
@@ -61,6 +61,7 @@ class GoalService(
       title = goal.title,
       areaOfNeed = areaOfNeedEntity,
       targetDate = goal.targetDate?.let { LocalDate.parse(it) },
+      reminderDate = goal.reminderDate?.let { LocalDate.parse(it) },
```

Comment:
Should we filter this so that the reminder date is only set if `status == GoalStatus.FUTURE` ?